server/app/services/languageProcessor/TaskMap.java

```diff
@@ -54,7 +56,11 @@ public JsonNode getTicketDescription(String issueKey, JsonNode responseBody) {
     if (Extractor.extractString(responseBody, "description").equals("[\"Issue Does Not Exist\"]")) {
       return parseToJson("fail", configuration.getString("error-message.issue-not-found"));
     } else {
-      String answer = "Description of " + issueKey + " is as follows: \n" +
+      String IssueId = responseBody.get("key").toString().replaceAll("\"", "");
+      String IssueUrl = "http://jira.agiledigital.com.au/browse/" + IssueId;
+      String Hyperlink = "<" + IssueUrl+ "|" + IssueId + ">";
```

Comment:
This code is the same as the code on line 80. This should be extracted out to a method, e.g. follow DRY principle.